config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -86,6 +86,59 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+  - name: pull-kubernetes-node-e2e-kubetest2
+    # explicitly needs /test pull-kubernetes-node-e2e to run
+    always_run: false
+    # if it is run and fails, don't block the PR
+    optional: true
+    branches:
+    - master
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    decoration_config:
+      timeout: 65m
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
```

Comment:
let's remove these two, from first glance they seem to be redundant in the old job as well since usually `preset-dind-enabled` is associated with `securityContext: privileged` and are required if we are running docker commands in the job.

we can add them later again if we run into issues.